plugins/Marketplace/vue/src/PluginList/PluginList.vue

```diff
@@ -299,6 +302,21 @@ export default defineComponent({
     openDetailsModal(plugin: Record<string, unknown>) {
       this.showPluginDetailsForPlugin = plugin;
     },
+    scrollPluginCardIntoView(plugin: Record<string, unknown>) {
+      const $titles = $(`.pluginListContainer .card-title:contains("${plugin.displayName}")`);
```

Comment:
Just a note — we could add a data attribute to the cards with the plugin name for easier selection than using the `:contains` selector with the display name, but this will do for now.